README.md

```diff
@@ -23,6 +23,12 @@ brew install railway
 npm i -g @railway/cli
 ```
 
+### curl
+
+```shell
+sh -c "$(curl -sSL https://github.com/railwayapp/cli/blob/master/install.sh)"
```

Comment:
This should point to the raw version, otherwise it downloads HTML
```suggestion
sh -c "$(curl -sSL https://raw.githubusercontent.com/railwayapp/cli/master/install.sh)"
```

![image](https://user-images.githubusercontent.com/3044853/101202637-6fc76c00-3661-11eb-8a00-7f0ffaa0b86f.png)